### PR TITLE
edit(README): Fix code block, small wording change

### PR DESCRIPTION
**Pull request by reeddunkle**
- Fix install command not rendering in code block
- Specify install options for clarity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,21 +98,24 @@ posh-git is available on the [PowerShell Gallery][psgallery-v1] and can be insta
...
+   depending on whether (A) you've never installed posh-git, or (B) if you've already installed a previous version:
 
-   ```powershell
-   # If you have never installed posh-git from the PowerShell Gallery
-   # NOTE: If asked to trust packages from the PowerShell Gallery, answer yes to continue installation of posh-git
-   # NOTE: If the AllowPrerelease parameter is not recognized, update your version of PowerShellGet to >= 1.6 e.g.
-   #       Install-Module PowerShellGet -Scope CurrentUser -Force -AllowClobber
-   PowerShellGet\Install-Module posh-git -Scope CurrentUser -AllowPrerelease -Force
-   ```
+```powershell
```
Comment (rkeithhill) [inline README.md]: Please indent the code-block like the previous one.

File: README.md [hunk 2]
```diff
@@ -98,21 +98,24 @@ posh-git is available on the [PowerShell Gallery][psgallery-v1] and can be insta
...
-   OR
+PowerShellGet\Install-Module posh-git -Scope CurrentUser -AllowPrerelease -Force
+```
 
-   ```powershell
-   # If you have already installed a previous version of posh-git from the PowerShell Gallery
-   PowerShellGet\Update-Module posh-git
+OR
+
+```powershell
```
Comment (rkeithhill) [inline README.md]: Ditto on the indent.

File: README.md [hunk 3]
```diff
@@ -98,21 +98,24 @@ posh-git is available on the [PowerShell Gallery][psgallery-v1] and can be insta
...
+# (B) You've already installed a previous version of posh-git from the PowerShell Gallery
+PowerShellGet\Update-Module posh-git
+```
```
Comment (rkeithhill) [inline README.md]: There are two sets of code-block closing tags. Need to remove one.

**Comment by reeddunkle** [comment]
You're welcome!
This is ready to review again

File: README.md [hunk 4]
```diff
@@ -98,22 +98,24 @@ posh-git is available on the [PowerShell Gallery][psgallery-v1] and can be insta
 1. Start either Windows PowerShell 5.x or PowerShell Core 6.x (`pwsh`).
 
 2. Execute one of the following two commands from an elevated PowerShell prompt,
-   depending on your current posh-git installation status:
+   depending on whether (A) you've never installed posh-git, or (B) if you've already installed a previous version:
```
Comment (dahlbyk) [inline README.md]: Nitpick: please amend without "if" after (B).

**Comment by reeddunkle** [comment]
- Removed "if"

This is ready to review again.

**Pull request merged**